params/computation_cost_params.go

```diff
@@ -78,6 +73,7 @@ const (
 	MloadComputationCost          = 376
 	MstoreComputationCost         = 288
 	Mstore8ComputationCost        = 5142
+	SloadComputationCost          = 835
```

Comment:
Some opcodes were set to apply the value of **EIP150** among EIP150 and EIP 1884, is there any reason for that?

Reply:
It is the revert of the computation cost modification which is applied at #958 :)